src/pint/scripts/event_optimize.py

```diff
@@ -411,6 +411,90 @@ def prof_vs_weights(self, nbins=50, use_weights=False):
             plt.savefig(ftr.model.PSR.value + "_htest_v_wgtcut_unweighted.png")
         plt.close()
 
+    def plot_priors(self, samples, weights=None, bins=100, scale=False, file=False):
+        """
+        Show binned samples, prior probability distribution and an initial gaussian
```

Comment:
We should be using the numpy docstring format for new functions and methods.  That is described here:
https://numpydoc.readthedocs.io/en/latest/format.html